app/src/ui/window/window-controls.tsx

```diff
@@ -34,7 +30,7 @@ interface IWindowControlState {
  */
 export class WindowControls extends React.Component<{}, IWindowControlState> {
   public componentWillMount() {
-    this.setState({ windowState: getWindowState(remote.getCurrentWindow()) })
+    this.setState({ windowState: 'normal' })
```

Comment:
Same about my other comment: maybe this should be initialized to `null` / `undefined`